### ISSUE TITLE
Queries with multiple e tags return no results on sqlite

### ISSUE THREAD

**Issue by bndw** [fiatjaf/relayer#47]
Single e tag works as expected
```
nostcat ws://localhost:9000
["REQ","RAND",{"kinds":[1],"#e":["8b1013c2d137bab5a7026e5adc65dcd641025c1a8ae2b7ecdec848cca44f1297"]}
```

Two values returns nothing:
```
nostcat ws://localhost:9000
["REQ","RAND",{"kinds":[1],"#e":["8b1013c2d137bab5a7026e5adc65dcd641025c1a8ae2b7ecdec848cca44f1297","e32f75bfce9513bab7c7e82b9a61fd56f80f68946a9d0a384f7f17989bd91f13"]}]
```

I suspect this has to do with the `LIKE` query generation. I will be working on a fix but if you can get it in quicker I'd gladly use that 😅

**Comment by fiatjaf** [comment]
The best fix would be to have a separate table for the tags to be queried.

What are you building? If you just want a simple sqlite-based relay for testing in the meantime I think you should probably use https://github.com/scsibug/nostr-rs-relay.

**Comment by bndw** [comment]
https://www.stemstr.app/

The backend is a relay + a couple endpoints for streaming stuff. I liked the idea of relayer because I can effectively embed the relay in the api and have hooks into the various relay lifecycles. Do you know of any other relay frameworks in Go?

**Comment by fiatjaf** [comment]
No, I don't, I don't know of any other relay frameworks in any language except for this one!

Wow, stemstr.app is looking great!, but hey, if that is your use case you should use the Postgres backend.

**Comment by bndw** [comment]
I was able to reproduce this problem using the Postgres backend.

**Comment by fiatjaf** [comment]
No way.

**Comment by fiatjaf** [comment]
Well, I guess we should fix it now then.

**Comment by bndw** [comment]
I apologize, after testing the Postgres backend again this morning I'm able to query by multiple `#e` tags. Too much debugging yesterday, sorry for the false alarm! 

Almost certain this _is_ a real issue in the sqlite backend.

**Comment by mattn** [comment]
The storage backends moved out of this repo into [fiatjaf/eventstore](https://github.com/fiatjaf/eventstore) — relayer now only takes an `eventstore.Store`, and there is no SQLite or Postgres query building left here. If this still reproduces it belongs in that repo. Closing.
